Makefile

```diff
@@ -1,25 +0,0 @@
-default: test
-
-detox-test:
-	detox
-
-travis-test: test
-
-test: env
-	.env/bin/py.test tests munch --doctest-modules
-
-coverage-test: env
-	.env/bin/coverage run .env/bin/nosetests -w tests
-
-env: .env/.up-to-date
```

Comment:
Why remove this? this is actually useful for development when mixing many different projects...